crates/sp1/evm-exec/script/src/bin/datagen.rs

```diff
@@ -1,61 +1,50 @@
 //! A script that generates proof input data for a range of EVM blocks.
```

Comment:
Most diffs are probably in this file which just scrapes docker compose services for proof inputs.

This previously relied on a hacked indexer from the ibc-demo that I used first time to get proof inputs. That's no longer the case now and uses metadata requests to the rollkit sequencer to get inclusion heights for data.